site/content/en/guides/consumer/function/catalog/_index.md

```diff
@@ -7,10 +7,6 @@ description: >
     Catalog of Config Functions.
```

Comment:
Remove this page and combine remaining two sentences into https://googlecontainertools.github.io/kpt/guides/consumer/function/

And revisit references to the catalog page from elsewhere on the site to see if they should link to top-level page or one of the new pages (https://googlecontainertools.github.io/kpt/search/?q=catalog)

Reply:
Done.